flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/StreamContextEnvironment.java

```diff
@@ -75,29 +62,31 @@ public JobExecutionResult execute() throws Exception {
 	@Override
 	public JobExecutionResult execute(String jobName) throws Exception {
 
-		JobGraph jobGraph;
-		if (jobName == null) {
-			jobGraph = this.getStreamGraph().getJobGraph();
-		} else {
-			jobGraph = this.getStreamGraph().getJobGraph(jobName);
-		}
-
-		transformations.clear();
-
-		// attach all necessary jar files to the JobGraph
-		for (URL file : jars) {
```

Comment:
I pushed a fix for this issue: bf29de981c2bcd5cb5d33c68b158c95c8820f43d
It think you have already adapted to this change. Nice :)